src/lib.rs

```diff
@@ -14,14 +14,17 @@
 pub mod rpc;
 pub mod utransport;
 
+use async_std::task::block_on;
+use crossbeam_channel::{unbounded, Receiver, Sender};
```

Comment:
I'm curious on the choice of using `crossbeam_channel` here over leaning on an async channel from `async_std`.

My guess is that since Zenoh is going to switch to using `tokio` in the 1.0 release you wanted something async framework-agnostic.

Reply:
The reason I used `crossbeam_channel` is that the receiving side is ThreadPool and no need to handle it as futures.